### ISSUE TITLE
Problem with merged fields from separate custom scripts

### ISSUE THREAD

**Issue by mahomedh** [netbox-community/netbox#4710]
### Environment
* Python version: 3.6.9
* NetBox version: 2.8.5

### Steps to Reproduce
1. Install the `create_vm.py` script and the `add_services.py` scripts (attached as `.txt` files)
[create_vm.py.txt](https://github.com/netbox-community/netbox/files/4718521/create_vm.py.txt)
[add_services.py.txt](https://github.com/netbox-community/netbox/files/4718522/add_services.py.txt)

2. click on create_vm
3. enter some parameters
4. click on Run Script
5. it rejects the entry and displays a whole load of parameters from the add_service script
6. viewing a fresh create_vm shows the merged fields

Only way to reset it is to restart the netbox service

### Expected Behavior
Only the fields for that specific script should show

### Observed Behavior
Running of either script results in merging of the fields of both scripts.

This has also been verified on the [mailing list](https://groups.google.com/d/msg/netbox-discuss/J0d6gBNPZKM/Pwa3ebJuBwAJ)

**Comment by DanSheps** [comment]
```
from extras.scripts import StringVar, Script
class Test1(Script):
    class Meta:
        name = "Test 1"
        description = "Test some stuff"
        field_order = [
            'vm_name',
        ]

    vm_name = StringVar(label="VM Name")
    def run(self, data, commit):
        self.log_success("Done Test 1")


class Test2(Script):
    class Meta:
        name = "Test 2"
        description = "Test some stuff"
        field_order = [
            'switch_name'
        ]

    switch_name = StringVar(label="Switch Name")

    def run(self, data, commit):
        self.log_success("Done Test 2")
```

Reproducible with the following code, to simplify testing.

**Comment by mahomedh** [comment]
> however you need to run one script and then the other script to get the reject.

Just to add to the above comment, you simply need to navigate to another script, then it's a bit random but it takes between 1 - 5 refreshes (without actually having to run the second script) to see the extra fields. The definite first step though is that you have to run one script then you cannot run any other script after that without a restart.

Please let me know if there's anything I can do to assist (including testing)

**Comment by jeremystretch** [comment]
This seems to be a byproduct of the change implemented under #4525 to allow scripts to accept initial data.
